frontend/src/views/CanteenEditor/SiretCheck.vue

```diff
@@ -190,8 +190,18 @@ export default {
       const canteenId = this.duplicateSiretCanteen.id
       return this.$store
         .dispatch("claimCanteen", { canteenId })
-        .then(() => (this.requestSent = true))
         .catch((e) => this.$store.dispatch("notifyServerError", e))
+        .then((response) => {
+          this.$store.dispatch("notify", {
+            status: "success",
+            message: "Vous gérez maintenant cette cantine.",
```

Comment:
```suggestion
            message: "Vous gérez maintenant cet établissement.",
```
Pinaillage... mais dans le cas d'une CC potentiellement ça sera plus clair ?